compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -144,6 +143,8 @@ void verifyTypeShape(const luci::CircleInput *input_node, const DataType &dtype,
   }
 }
 
+// TODO Remove unused code
+#if 0
 void update_quantparam(record_minmax::MinMaxObserver *observer, const std::string &mode,
```

Comment:
## Motivation behind this PR

Previously, this local function computes min/max values for all algorithms. This makes it is difficult to update an algorithm, e.g., adding more hyperparameters (#11269).

This PR introduces MinMaxComputer to tackle the issue. After this PR, we may only update MinMaxComputer to change min/max computation logic.